scripts/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-use-before-define */
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const list = document.getElementById('list');
+
+let books = [];
+
+function addingBookToHtml(Title, Author, index) {
+  const div = document.createElement('div');
+  div.className = 'list';
+  div.setAttribute('id', index);
+  const title = document.createElement('p');
+  title.innerHTML = Title;
+  title.className = 'bookTitle';
+  const author = document.createElement('p');
+  author.innerHTML = Author;
+  author.className = 'bookAuthor';
+  const button = document.createElement('button');
+  button.innerHTML = 'Remove';
+  button.className = 'RemoveButton';
+  button.setAttribute('id', 'removeButton');
+
+  button.addEventListener('click', ((e) => {
+    const parent = e.target.parentElement;
+    const index = parent.id;
+    books.splice(index, 1);
```

Comment:
- [OPTIONAL] I highly suggest that you use the filter method because it does not mutate the array:
  - Immutable values are always good for performance and are a best practice
  - You can do something like the following: `books = books.filter((book) => book.id !== index)`, But make sure you add the id property in the book object when first create it

Reply:
- [ ] [Optional] just as the previous reviewer mention, it will be nice if you use filter method :+1: